### DISCUSSION TITLE
.NET MAUI  (.NET 8) really frustating on Mac

### DISCUSSION THREAD

**Discussion by dharamhbtik** [dotnet/maui#20507] [General]
Today I tried to create a simple MAUI project using .NET 8 and just tried to run without making any changes, it started giving the following error and build status as succeeded.  

When I am running for MacOS, it gives the below error and starts the Hello World App.

When I am running for Android 14 (Pixel 5) it keeps deploying to the device for more than 30 minutes, Stopped, deleted the bin and obj folder tried to run on a real device, but it again stuck on the same and displayed the same error as below.

It is frustrating to develop a simple app in MAUI. How can Microsoft expect an enterprise app in MAUI?  Will it be like their other products such as LightSwitch?

I think switching to other cross-platform frameworks such as Flutter or Kotlin multiplatform will be easier than .NET MAUI. 

I will have a look at Windows to see how it is behaving, as Microsoft is going to discontinue VS for Mac and encourage to us of VS code on Mac for MAUI development. Will see how VS code will treat MAUI.

What do you think?

<img width="1374" alt="image" src="https://github.com/dotnet/maui/assets/8122751/7b31d2a9-5e6b-4dad-b909-b38408e93cd2">

**Comment by bcaceiro** [comment]
The experience on Visual studio Code is horrible. I wonder how people add new Files, and other simple stuff like this, since there are no templates, no correct Intellissense, and VSMac is just dead.
